assets/_sass/base/_typography.scss

```diff
@@ -65,6 +75,27 @@ a {
   }
 }
 
+// TODO audit codebase to determine if this works. Remove link-alt if it works.
+/*
+h1, h2, h3, h4, h5, h6 {
+  > a {
+    color: $red;
+    text-decoration: none;
+    &:hover, &:focus {
+        text-decoration: underline;
+    }
+  }
+}
+*/
+
+.link-alt {
+  color: $red;
+  text-decoration: none;
+  &:hover, &:focus {
```

Comment:
Rule declaration should be preceded by an empty line<br>Each selector in a comma sequence should be on its own line